ui-test/common/utils.ts

```diff
@@ -0,0 +1,75 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import { Workbench, VSBrowser, BottomBarPanel, InputBox } from 'vscode-extension-tester';
+import { notificationCenterIsOpened, inputHasNewMessage, terminalHasText} from './conditions';
+
+export async function loginToOpenShiftWithTerminal(clusterUrl: string, username: string, password: string): Promise<void> {
+  const terminalView = await new BottomBarPanel().openTerminalView();
+  await terminalView.executeCommand('rm -f .kube/config');
+  await terminalView.executeCommand('oc login --username=' + username + ' --password=' + password + ' --server=' + clusterUrl + ' --insecure-skip-tls-verify');
+  await terminalView.executeCommand('oc new-project test-tekton');
+  await terminalHasText(terminalView, 'Login successful.');
+  await terminalView.killTerminal();
+}
+
+export async function logoutFromOpenShiftWithTerminal(): Promise<void> {
+  const bottomBar = new BottomBarPanel();
+  await bottomBar.toggle(true);
+  const terminalView = await bottomBar.openTerminalView();
+  await terminalView.executeCommand('oc delete project test-tekton');
+  await terminalView.executeCommand('oc logout');
+  await terminalHasText(terminalView, 'Logged');
+  await terminalView.killTerminal();
+}
+
+export async function clearNotifications(): Promise<undefined> {
```

Comment:
This could be a `Promise<void>`, then it would work without the `return null` line.